bin/logstash.lib.sh

```diff
@@ -1,4 +1,9 @@
-basedir=$(cd `dirname $0`/..; pwd)
+unset CDPATH
```

Comment:
I am not sure, it was like this before and I haven't tested the effect of removing it.